lib/util/http/github.ts

```diff
@@ -152,8 +152,11 @@ function constructAcceptString(input?: any): string {
 }
 
 export class GithubHttp extends Http<GithubHttpOptions, GithubHttpOptions> {
-  constructor(options?: GithubHttpOptions) {
-    super(PLATFORM_TYPE_GITHUB, options);
+  constructor(
+    options?: GithubHttpOptions,
+    hostType: string = PLATFORM_TYPE_GITHUB
+  ) {
+    super(hostType, options);
   }
```

Comment:
Can we flip the order? We never passed any `options` yet, so `hostType` is more mandatory and should com first.

We can refactor this in a later pr for the other HTTP classes